docs/getting_started.rst

```diff
@@ -86,6 +86,9 @@ Use the ``HTTPAdapter`` argument ``pool_connections`` to set the number of
 urllib3 connection pools to cache, and the ``pool_maxsize`` argument to set the
 maximum number of connections to save in the pool.
 
+As long as the state of ``client`` does not changed it will be
+thread safe and accessible as a static variable to all threads.
+
```

Comment:
@snowch I'm going to remove this. The `requests toolbelt` library (and the comment from Sam) says that although `requests` is documented as theadsafe there are cases where it's not perfectly threadsafe: http://toolbelt.readthedocs.io/en/latest/threading.html.  I think it's best to [add a comment that recommends one `client` per thread.](https://github.com/cloudant/python-cloudant/pull/272#discussion_r95386901)